spec.html

```diff
@@ -542,8 +541,53 @@ <h1>[[Construct]] ( _argumentsList_, _newTarget_)</h1>
 <emu-clause id="sec-private-names">
   <h1>Private Names and references</h1>
 
-  <p>Each PrivateName value immutably holds a mutable record called [[Attributes]], initially empty, which may have the following fields added to it:</p>
-    <emu-table id="private-name-record" caption="Fields of the [[Attributes]] record">
+  <p>The <dfn>Private Name</dfn> specification type is used to describe a globally unique record which represents a private <del>field</del><ins>class element (field, method, or accessor)</ins>. A Private Name may be installed on any ECMAScript object with the PrivateFieldAdd internal algorithm, and then read or written using PrivateFieldGet and PrivateFieldSet.</p>
```

Comment:
If we're saying that Private Name is already a record, I guess there's no need to separate out the `[[Attributes]]` into a separate sub-record. Let's make this change in the same patch, since you'd need to update the users of Private Name records for this intermediate state.

Reply:
Ah, good. Will do. I think just private methods and decorators are affected by it, but I'll double-check.

Reply:
Looks great, thanks!